scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -29,16 +30,44 @@ namespace <%= namespace %>.SDK.iOS {
             var contentDictionary = (Dictionary<string, object>)Json.Deserialize(message.Content);
             var mailingAddressInput = new MailingAddressInput(contentDictionary);
 
-            List<SummaryItem> summaryItems = null;
-            List<ShippingMethod> shippingMethods = null;
+            Action respondWithSuccess = () => {
+                message.Respond(new ApplePayEventResponse(ApplePayAuthorizationStatus.Success, GetSummaryItems(), GetShippingMethods()).ToJsonString());
+            };
+
+            Action<ShopifyError> respondToError = (ShopifyError error) => {
+                switch(error.Type) {
+                case ShopifyError.ErrorType.GraphQL:
+                case ShopifyError.ErrorType.HTTP:
+                    message.Respond(new ApplePayEventResponse(ApplePayAuthorizationStatus.Failure).ToJsonString());
+                    break;
+                case ShopifyError.ErrorType.UserError:
+                    var userErrors = CurrentCart.UserErrors;
+
+                    if (userErrors.Count > 0) {
```

Comment:
Is there a case where a `ShopifyError.ErrorType.UserError` would be created however `userErrors.Count == 0`